src/lwc/geFormRenderer/__tests__/data/dataImportObjectInfo.json

```diff
@@ -0,0 +1,5200 @@
+{
```

Comment:
How do you generate this - just building a DI out on the platform and saving off the serialization? I'm curious how much ongoing maintenance we might have to do, if we add fields to DI in the future?

Reply:
We'd need to update the dataImportMock whenever we update the Template Mock to include a new field, because we use this mock to check for CRUD access to fields on the template.